backend/sozisel/lib/sozisel/model/polls/poll_result.ex

```diff
@@ -18,4 +19,58 @@ defmodule Sozisel.Model.Polls.PollResult do
     |> cast(attrs, [:option_ids])
     |> validate_required([:option_ids])
   end
+
+  @spec poll_summary(Ecto.UUID.t()) :: map() | nil
+  def poll_summary(launched_event_id) do
+    alias Sozisel.Repo
```

Comment:
aliases should be put outside the function at the module scope (Ithink that I may have done it but in the end it looks ugly)